modules/nf-core/bismark/coverage2cytosine/main.nf

```diff
@@ -9,7 +9,7 @@ process BISMARK_COVERAGE2CYTOSINE {
 
     input:
     tuple val(meta), path(coverage_file)
-    tuple val(meta2), path(fasta)
+    tuple val(meta2), path(fasta, stageAs: 'tmp/*')
```

Comment:
Same here. A comment would be helpful explaining why this is staged but not used directly.

Reply:
done